Sources/SwiftDocC/Semantics/ExternalLinks/ExternalReferenceWalker.swift

```diff
@@ -174,21 +174,28 @@ struct ExternalReferenceWalker: SemanticVisitor {
     }
 
     mutating func visitComment(_ comment: Comment) { }
-    
+
+    mutating func visitMarkups(_ markups: [Markup]?) {
```

Comment:
I've replace `visitMarkups` with `visitSection` as you suggested and also added `visitSectionVariants` to avoid some repetition. This uses the `some` keyword so we will have to wait until you can merge #861 to merge this.